lib/std/paths.nim

```diff
@@ -17,6 +20,9 @@ export ReadDirEffect, WriteDirEffect
 type
   Path* = distinct string
 
+func `==`*(x, y: Path): bool {.inline.} =
```

Comment:
No! Equality for Paths can be quite complex. At least you need to ignore case when on Windows (roughly) and you also need to offer a `hash` implementation.

Reply:
There is also some logic somewhere for path canonicalization that needs to be used here.

Reply:
Sure, I will use `cmpPath`.